gammapy/irf/effective_area.py

```diff
@@ -0,0 +1,71 @@
+#Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ..utils.nddata import NDDataArray, DataAxis, BinnedDataAxis
+from collections import OrderedDict
+import numpy as np
+
+class EffectiveArea2D(NDDataArray):
+    """2D effective area table
+
+    Axes: ``THETA``, ``ENERG``
```

Comment:
The axis names are currently just the column names in the Fits file, we might want to map this to ``energy`` and ``offset``?